CONTRIBUTING.md

```diff
@@ -224,20 +224,20 @@ You're awesome. :wave::smiley:
 [sphinx]: http://www.sphinx-doc.org/en/master/index.html
 [readthedocs]: https://docs.readthedocs.io/en/latest/index.html
 
-[link_issues]: https://github.com/poldracklab/FMRIPREP/issues
-[link_labels]: https://github.com/poldracklab/FMRIPREP/labels
+[link_issues]: https://github.com/poldracklab/fMRIPrep/issues
+[link_labels]: https://github.com/poldracklab/fMRIPrep/labels
 [link_discussingissues]: https://help.github.com/articles/discussing-projects-in-issues-and-pull-requests
 
-[link_bugs]: https://github.com/poldracklab/FMRIPREP/labels/bug
-[link_helpwanted]: https://github.com/poldracklab/FMRIPREP/labels/help%20wanted
-[link_feature]: https://github.com/poldracklab/FMRIPREP/labels/feature
+[link_bugs]: https://github.com/poldracklab/fMRIPrep/labels/bug
+[link_helpwanted]: https://github.com/poldracklab/fMRIPrep/labels/help%20wanted
+[link_feature]: https://github.com/poldracklab/fMRIPrep/labels/feature
 
-[link_style_guide]: https://github.com/poldracklab/fmriprep/blob/master/CONTRIBUTING.md#fmriprep-coding-style-guide
+[link_style_guide]: https://github.com/poldracklab/fMRIPrep/blob/master/CONTRIBUTING.md#fMRIPrep-coding-style-guide
 [link_pullrequest]: https://help.github.com/articles/creating-a-pull-request/
 [link_fork]: https://help.github.com/articles/fork-a-repo/
 [link_pushpullblog]: https://www.igvita.com/2011/12/19/dont-push-your-pull-requests/
 [link_branches]: https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/
 [link_updateupstreamwiki]: https://help.github.com/articles/syncing-a-fork/
 [link_stemmrolemodels]: https://github.com/KirstieJane/STEMMRoleModels
-[link_zenodo]: https://github.com/poldracklab/FMRIPREP/blob/master/.zenodo.json
-[link_devel]: http://fmriprep.readthedocs.io/en/latest/contributors.html
+[link_zenodo]: https://github.com/poldracklab/fMRIPrep/blob/master/.zenodo.json
+[link_devel]: http://fMRIPrep.readthedocs.io/en/latest/contributors.html
```

Comment:
Generally should be lowercase in URLs, but I don't think any of these are going to be problems. Could you go ahead and make this one an `https` link?

Reply:
Ah sorry, I did a lazy replace-all. Patching now and I'll switch to `https` !